### ISSUE TITLE
problem in implementation 

### ISSUE THREAD

**Issue by ayushiagrahari95** [wstrange/GoogleAuth#56]
I have dropped the created jar in the installation in opt/alfresco_installation/tomcat/lib folder but nothing happens.No extra field for capturing the passcode appears on the alfresco share page except the username and password.
I have also tried by just dropping the dependencies in my repo and share maven projects.But still nothing happens.

and I am little confused that this GoogleAuth is just the code for the backened validation that creates and verifies OTPs or is it the complete code that has to be integrated with our maven code?what about the UI part.how to render the extra input field in alfresco share page.
Basically.I am confused with its complete implementation in alfresco.

Any help would be greatly appreciated.

**Comment by emcrisostomo** [comment]
Nothing happens because this is not an Alfresco plugin, nor it provides any UI. As the repository description says, this is an implementation of the TOTP algorithm in Java. No more, no less.

**Comment by ayushiagrahari** [comment]
Okie that's what I had understood but was little confused about it.Well,thanks for that.

Now,can u please tell me where to actually find the code for UI so that the third input field for enteribg the otp appears on the alfresco share page and 2 factor authentication would start working.

**Comment by emcrisostomo** [comment]
No, I can't.
